src/celeritas/CMakeLists.txt

```diff
@@ -164,6 +164,28 @@ if(CELERITAS_USE_CUDA OR CELERITAS_USE_HIP)
   )
 endif()
 
+if(CELERITAS_USE_covfie)
+  set(_celer_covfie_src
+    field/CartMapFieldParams.covfie.cc
+  )
+  celeritas_polysource_append(_celer_covfie_src
+    alongstep/AlongStepCartMapFieldMscAction)
+
+  celeritas_add_object_library(celeritas_covfie ${_celer_covfie_src})
+  celeritas_target_link_libraries(celeritas_covfie
+    PRIVATE
+      Celeritas::BuildFlags
+      Celeritas::geocel
+      ${_core_geo_deps}
+      Celeritas::Extcovfie
+  )
+  list(APPEND SOURCES $<TARGET_OBJECTS:celeritas_covfie>)
+  list(APPEND PRIVATE_DEPS celeritas_covfie)
```

Comment:
It seems that the cxx20 requirement isn't being propagated in CUDA build. I have to explicitly set `CMAKE_CXX_STANDARD=20`.  the `.cu` file is built with `-std=c++17` while the .cc file is built with `-std=c++20`.

Reply:
Arggg @pcanal ..... ok sorry about that Julien, I'll see if I can fix right quick.

Reply:
OK actually the issue is https://github.com/acts-project/covfie/pull/78  not RDC. It seems like the build passes? I have to update my dev toolchain to get CUDA with C++20 so it may be a bit.

Reply:
very nice, thanks for the final fixes!